contracts/ETO/ETOTokenTerms.sol

```diff
@@ -73,8 +77,7 @@ contract ETOTokenTerms is Math, IContractId {
         require(keccak256(abi.encodePacked(equityTokenName)) != EMPTY_STRING_HASH);
         require(keccak256(abi.encodePacked(equityTokenSymbol)) != EMPTY_STRING_HASH);
         // overflows cannot be possible
-        require(maxNumberOfTokens < 2**56, "NF_TOO_MANY_TOKENS");
-        require(mul(tokenPriceEurUlps, maxNumberOfTokens) < 2**112, "NF_TOO_MUCH_FUNDS_COLLECTED");
+        require(maxNumberOfTokens < 2**128, "NF_TOO_MANY_TOKENS");
```

Comment:
what is the reasoning behind increasing to 2**128 here? 

if we want to keep the same limit... it would be something around 2**64, i think...

Reply:
storage comes in 256bits chunks so increasing to 64bits allocates another WORD anyway. so we can go with 128bits and still use same amount of gas.

what do you mean by the same limit? if the token is 18 decimals then you need log2(10**18) ~59 bits to store one token and that is the increase

Reply:
you're correct. I dont rmember how i got to that result. we need to have 2**56 * 2**59, so 128 is the next meaningful size, i agree